src/components/select-project/select-project.component.ts

```diff
@@ -4,6 +4,7 @@ export const selectProject: angular.IComponentOptions = {
   bindings: {
     selectedProject: '=',
     nameTaken: '<',
+    onNewProjectNameChanged: '&',
```

Comment:
I don't adding a new callback is necessary... The embedding page can just watch `selectedProject.metadata.name`.

```ts
this.$scope.$watch('$ctrl.selectedProject.metadata.name', () => {
  this.ctrl.projectNameTaken = false;
});
```

Reply:
Fixed.